src/components/ButtonMenu.stories.tsx

```diff
@@ -157,3 +157,21 @@ export function WithTooltip() {
 
   return <ButtonMenu trigger={{ label: "Menu trigger" }} items={menuItems} tooltip="Tool tip text" />;
 }
+
+export function WithSearchListItems() {
+  const menuItems: MenuItem[] = [
+    { label: "Development 1 - Cohort 1", onClick: action("Development 1 - Cohort 1") },
+    { label: "Development 1 - Cohort 2", onClick: action("Development 1 - Cohort 2") },
+    { label: "Development 1 - Cohort 3", onClick: action("Development 1 - Cohort 3") },
+    { label: "Development 2 - Cohort 1", onClick: action("Development 2 - Cohort 1") },
```

Comment:
quibble (non-blocking): These examples didn't make it really easy to test the search. The only thing one could use to get options to change was the numbers. 🤷

Reply:
True, let me make them more distinct